IPython/sphinxext/ipython_directive.py

```diff
@@ -294,14 +294,15 @@ def __init__(self, exec_lines=None):
         IP = InteractiveShell.instance(config=config, profile_dir=profile)
         atexit.register(self.cleanup)
 
-        # io.stdout redirect must be done after instantiating InteractiveShell
-        io.stdout = self.cout
-        io.stderr = self.cout
+        # stdout redirect must be done after instantiating InteractiveShell
+        # FIXME: is that still true in IPython 5?
```

Comment:
I don't think so; for `io.stdout` we needed to because instantiating `InteractiveShell` set `io.stdout` to something else. I don't think initialising the shell changes `sys.std*`.